json_schema/python/validate.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import yaml
+from pathlib import Path
+from jsonschema import validate, validators
```

Comment:
How about adding at least `requirements.txt`? 😉

Reply:
Done, i've added some details to a readme file as well to get the test running :)